static/sass/styles.scss

```diff
@@ -1099,3 +1100,81 @@ hr.p-separator.is-shallow {
 .no-border::after {
   content: none !important;
 }
+
+// Custom styles /dell
+$dell-blue: #232943;
+.p-strip--dell-blue {
+  @extend .p-strip--dark;
+
+  background-color: $dell-blue;
+}
+
+.p-card--dell-webinars {
+  background: #000
+    url("https://assets.ubuntu.com/v1/4329b93d-vertical-thumb_111.png") right;
+  color: #fff;
+  height: 400px;
+  padding: calc(1rem - 1px);
+
+  @media only screen and (max-width: $threshold-6-12-col) {
+    background-color: #000;
+    background-size: cover;
+    height: auto;
+    margin-bottom: 1.5rem;
+  }
+
+  .p-icon--video-play {
+    color: #fff;
+    height: 28px;
+    width: 28px;
+  }
+}
+
+.p-aspect-ratio-image {
+  height: 0;
+  padding-top: calc(200% / 3);
+  width: 100%;
+}
+
+.p-aspect-ratio-image--desktop-solutions {
+  @extend .p-aspect-ratio-image;
+
+  background: #666
+    url("https://assets.ubuntu.com/v1/3c7f0508-DELLBartonGe_3-1661193946868.png")
```

Comment:
We should add media queries to these images to hide them on small and medium screens

Reply:
@lyubomir-popov correct me if I am wrong, with this new way of adding images, we are supposed to have these images showing in small screens too